src/common/modal/MuiModal.jsx

```diff
@@ -0,0 +1,30 @@
+import { Modal, Box } from "@mui/material";
```

Comment:
I don't think this component is being used.

Reply:
removed